x-pack/solutions/observability/plugins/apm/public/components/app/onboarding/instructions/otel_agent.tsx

```diff
@@ -39,34 +38,41 @@ export const createOpenTelemetryAgentInstructions = (
   return [
     {
       title: i18n.translate('xpack.apm.onboarding.otel.download.title', {
-        defaultMessage: 'Download the OpenTelemetry APM Agent or SDK',
+        defaultMessage: 'Instrument your app with EDOT SDK ',
```

Comment:
Should we clarify for users what EDOT means? `Elastic Distributions of OpenTelemetry`

Reply:
Should non serverless environment show the same set of instructions?

Reply:
Got it. I got confused because we have a function and a react component with the same name `createOpenTelemetryAgentInstructions`. The function is used on the Cloud and onprem onboarding and the react component is used on serverless onboarding.

Reply:
Yes true, this confused me at first as well